manifest.json

```diff
@@ -3,14 +3,29 @@
     "name": "GitHub Story Points",
     "description": "Chrome extension to see story points in GitHub projects",
     "short_name": "GitHub Story Points",
-    "version": "0.2.5",
+    "version": "0.2.7",
     "author": "Kohei Hasegawa",
     "homepage_url": "https://github.com/banyan/github-story-points",
     "icons": {
         "16": "assets/images/icon.png",
         "48": "assets/images/icon.png",
         "128": "assets/images/icon.png"
     },
+    "permissions": [
+        "https://github.com/*",
+        "contextMenus",
+        "activeTab"
+    ],
+    "optional_permissions": [
+        "http://*/*",
+        "https://*/*"
+    ],
+    "background": {
+        "scripts": [
+            "dist/background.js"
+        ],
+        "persistent": false
+    },
```

Comment:
If you want Firefox to show the icon you'll have to specify a `browser_action` in here as well:

```json
	"browser_action": {
		"default_icon": "icon.png"
	},
```

Reply:
Thanks for the review too! 🙇

>If you want Firefox to show the icon you'll have to specify a browser_action in here as well:

Does the icon correspond to what you say?

![image](https://user-images.githubusercontent.com/19625/62818649-fc087280-bb84-11e9-9f29-218880af8a43.png)

if so, it shows without `browser_action`? 🤔

Reply:
> if so, it shows without `browser_action`? 🤔

Oh really? They must have changed something.